src/main/java/io/obswebsocket/community/client/OBSRemoteController.java

```diff
@@ -567,4 +567,20 @@ public void toggleRecordPauseRequest(Consumer<ToggleRecordPauseResponse> callbac
     public void toggleRecordRequest(Consumer<ToggleRecordResponse> callback) {
```

Comment:
Just noticed in here and in other places -- IMO the methods on the remote should be of the form "getSystemStats", "getProjectorList", etc rather than including 'Request' in it.

What do you think @ChristopheCVB ?

Reply:
I guess we could do that yeah, but that's just cosmetics. I don't really have an opinion on this.

Reply:
It's a noun-verb correctness thing I guess
"toggleRecordRequest" for example would have the verb "toggle" and noun "RecordRequest". 
It's close, but I guess technically we're toggling a "Record" and not a "RecordRequest".

And if we look at the other methods, half of them follow this convention and the others don't; we should probably stick a convention keep to it.

Reply:
Cosmetically, we could go either way -- but if we go the way here, then maybe we should update the other methods to follow the same conventions?

Reply:
Either way, not something we need to worry about in this particular PR